#
# This file is autogenerated by pip-compile with Python 3.10
# by the following command:
#
#    pip-compile requirements/dev-requirements.in
#
attrs==25.3.0
    # via
    #   jsonschema
    #   referencing
black==25.1.0
    # via -r requirements/dev-requirements.in
certifi==2025.6.15
    # via requests
charset-normalizer==3.4.2
    # via requests
click==8.1.8
    # via
    #   -r requirements/requirements.in
    #   black
et-xmlfile==2.0.0
    # via openpyxl
exceptiongroup==1.3.0
    # via pytest
flake8==7.2.0
    # via -r requirements/dev-requirements.in
idna==3.10
    # via requests
iniconfig==2.1.0
    # via pytest
isort==6.0.1
    # via -r requirements/dev-requirements.in
jsonschema==4.25.1
    # via -r requirements/requirements.in
jsonschema-specifications==2025.9.1
    # via jsonschema
mccabe==0.7.0
    # via flake8
mypy-extensions==1.1.0
    # via black
openpyxl==3.1.5
    # via -r requirements/requirements.in
packaging==25.0
    # via
    #   black
    #   pytest
pathspec==0.12.1
    # via black
platformdirs==4.3.8
    # via black
pluggy==1.6.0
    # via pytest
pycodestyle==2.13.0
    # via flake8
pyflakes==3.3.2
    # via flake8
pygments==2.19.2
    # via pytest
pytest==8.4.1
    # via -r requirements/dev-requirements.in
python-dotenv==1.1.1
    # via -r requirements/requirements.in
python-frontmatter==1.1.0
    # via -r requirements/requirements.in
pyyaml==6.0.2
    # via python-frontmatter
referencing==0.36.2
    # via
    #   jsonschema
    #   jsonschema-specifications
requests==2.32.4
    # via -r requirements/requirements.in
rpds-py==0.27.1
    # via
    #   jsonschema
    #   referencing
tomli==2.2.1
    # via
    #   black
    #   pytest
typing-extensions==4.15.0
    # via
    #   black
    #   exceptiongroup
    #   referencing
urllib3==2.5.0
    # via requests
